### DISCUSSION TITLE
Federated binding missing elements

### DISCUSSION THREAD

**Discussion by balverio1020** [CoreWCF/CoreWCF#687] [Q&A]
Is there a way to set the AppliesTo on the CoreWcf federated binding? I need this element and I dont see a way to set it

**Comment by mconnew** [comment]
WSFederationHttpBinding in .NET Framework doesn't have an `AppliesTo` property. Can you provide the code snippet how you set this on .NET Framework?

**Comment by balverio1020** [comment]
for .net you use the token parameters that gets set inside of the binding as shown below:

var rst = new RequestSecurityToken
                {
                    RequestType = RequestTypes.Issue,
                    KeyType = KeyTypes.Bearer,
                    TokenType = "http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0",
                    AppliesTo = new EndpointReference("http://myrealm"),
                    Issuer = new EndpointReference("https://localhost/adfs/services/trust/13/usernamemixed")
                };

but for net core i cant find a way to do this and i need to for my companies new product so that we can be backwards compatible

**Comment by mconnew** [comment]
That's for the client side requesting a security token from an STS using WS-Trust. If this is a client scenario you are looking at, I think you want https://github.com/dotnet/wcf
